layouts/documentations/documentations.html

```diff
@@ -0,0 +1,15 @@
+{{ partial "header.html" . }}
```

Comment:
I thought we have a default layout that looks like this one that would be used instead, less duplication that way.

Reply:
How do I use the default layout? If I don't make a documentations.html file, the content doesn't seem to follow the default layout by itself.

Reply:
Then perhaps we do not have one :(

Reply:
There does exist a _default in /layouts folder. But I don't know how to use that layout here. Can you please guide me on how to link a layout to a markdown file?

OR, should I just make a documentation.html file and write both content + layout (html) as suggested above?

Reply:
Have a look at the [template lookup order](https://gohugo.io/templates/lookup-order/) in Hugo documentation. For `_index.md` files I think `layouts/_default/section.html` is used.